### PR TITLE
chore: remove CODEOWNERS

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR removes codeowners.

### Context of Change

@ledhed2222 and I are no longer the owners of this repo

### Type of Change

- [x] Repo management

## Test Plan

Code isn't touched.

### CODE REVIEW

File: CODEOWNERS
Comment (ledhed2222) [inline CODEOWNERS:1]: shouldn't we replace with new owners? if none defined, doesn't seem bad to me to keep this until there are new ones defined

Comment (mvadari) [reply CODEOWNERS:1]: We don't have any codeowners in the xrpl.js repo and it's fine - IMO this isn't really necessary for these repos.

Comment (mvadari) [reply CODEOWNERS:1]: Unless you think it'll be actually useful given that different groups are going to be independently working in these repos? Either way, the decision should be the same across the two repos we control(led).

**Review by ledhed2222 — APPROVED** [review]

**Review by JST5000 — APPROVED** [review]
I think it makes sense to not have centralized code owners - Just getting a review from whoever committed that part of the code last/the maintainers makes sense to me.

**Comment by mvadari** [comment]
> I think it makes sense to not have centralized code owners - Just getting a review from whoever committed that part of the code last/the maintainers makes sense to me.

The objective of codeowners is to represent those maintainers - basically, when this was in place, you had to get an approval from @ledhed2222 or I in order to merge your PR.

**Comment by JST5000** [comment]
> The objective of codeowners is to represent those maintainers - basically, when this was in place, you had to get an approval from @ledhed2222 or I in order to merge your PR.

I see, if it becomes a problem, we'll re-implement it, but I don't think it's necessary for now :)

**Pull request merged**
